.github/workflows/c3-dependabot-versioning-prs.yml

```
@@ -32,4 +32,4 @@ jobs:
           git config --global user.name 'Wrangler automated PR updater'
 
       - name: Generate C3 changesets
-        run: node -r esbuild-register tools/dependabot/generate-dependabot-pr-changesets.ts ${{ github.event.number }} create-cloudflare packages/create-cloudflare/src/frameworks/package.json
+        run: node -r esbuild-register tools/dependabot/generate-dependabot-pr-changesets.ts ${{ github.event.number }} create-cloudflare packages/create-cloudflare/src/frameworks/package.json c3-frameworks-update
```

shall we also add a comment to try to avoid this getting updated in the future without also updating the other workflow?
```suggestion
        # Note: the c3-frameworks-update prefix needs to be such as it is used by the .github/workflows/c3-e2e-dependabot.yml workflow
        run: node -r esbuild-register tools/dependabot/generate-dependabot-pr-changesets.ts ${{ github.event.number }} create-cloudflare packages/create-cloudflare/src/frameworks/package.json c3-frameworks-update
```

---

Good idea.